analysis/paladinholy/src/CONFIG.tsx

```diff
@@ -10,7 +10,7 @@ import CHANGELOG from './CHANGELOG';
 const config: Config = {
   contributors: [acornellier],
   expansion: Expansion.Shadowlands,
-  patchCompatibility: '9.0.5',
```

Comment:
Are we really ready for 9.2 support?

Reply:
From things I've heard from Ferbian and others, maraads implementation was the only thing missing.